src/js/components/FormField/__tests__/__snapshots__/FormField-test.js.snap

```diff
@@ -561,7 +561,6 @@ exports[`FormField disabled 1`] = `
               onChange={[Function]}
               onFocus={[Function]}
               onKeyDown={[Function]}
-              value=""
```

Comment:
looking at the change in line 148 FormField.js, I'm wondering how it is possible this line got removed? 
I'm asking because last time we removed the default string value from snapshots we had many issues coming regarding backward compatibility.

Reply:
Do you recall any. more detail about what the issues were?

Reply:
I think those: (might be more)
https://github.com/grommet/grommet/pull/3874
https://github.com/grommet/grommet/issues/3879

Reply:
I'm glad you pointed these out. I re-read https://reactjs.org/docs/forms.html#controlled-input-null-value and realized why we need to set a value. Fixed.